src/__tests__/entities/currency.test.ts

```diff
@@ -1,4 +1,4 @@
-import { Ether, Token } from './index'
+import { Ether, Token } from 'src/entities/index'
```

Comment:
can you put the test files back where they were? it's unrelated to the changes in the PR, we can discuss it in a separate PR if you think it's an improvement

Reply:
The changes to the core sdk and v3 SDK were developed and organized in a Uniswap Foundation fork as this is part of a grant.
We had several pull requests with separate features, but this upstream pull request is the completed milestone 1.

I really don't want to split it apart again just for the sake of having multiple PRs. The grant and its milestones were approved by both the Foundation and Labs. As long as we don't have push access to the upstream branch to continue development in branches directly here, there is no way for us to efficiently work other than like this. Especially since getting reviews on individual PRs seems to take 2-3 months.

Reply:
I agree with @just-toby, would you mind expanding on the issue once a bit more @koraykoska? I understand there's some interoperability issue due to the way things are set up, but I have trouble getting what's the actual issue here.

Either way, it would be great to resolve, because our standardised library structure ships with inline test files and moving to `__tests__` is going to be confusing in the long-run.

Reply:
@grabbou Inline tests are simply not a scalable solution and nearly no modern language or design guide allows them. This is a public and open-source repository that is meant to be easy to read through and understand, not part of the Uniswap internal codebase. It's expected to grow even further in the scope of our grant and with the additions for V4 (and maybe Uniswap X stuff?). It's not helpful to revert changes to an obviously inferior version just for the sake of separating PRs into functionalities.
Going forward we will submit PRs based on specific features to make review turnaround times faster, but I don't see a reason to revert this now.